lib/local-cluster/lib/Cardano/Wallet/Launch/Cluster/Monitoring/Http/Client.hs

```diff
@@ -1,56 +1,45 @@
-{-# LANGUAGE DataKinds #-}
 {-# LANGUAGE DerivingStrategies #-}
-{-# LANGUAGE GADTs #-}
-{-# LANGUAGE NamedFieldPuns #-}
-{-# LANGUAGE NumericUnderscores #-}
-{-# LANGUAGE RankNTypes #-}
-{-# LANGUAGE RecordWildCards #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE LambdaCase #-}
 
 module Cardano.Wallet.Launch.Cluster.Monitoring.Http.Client
```

Comment:
```suggestion
module Cardano.Wallet.Launch.Cluster.Http.Client
```

This module combines both the `Application`/`Faucet` and the `Monitoring`, doesn't it? 🤔